js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,14 @@
+// import { isString } from "util";
+
+function abbreviateString(str) {
+    if (typeof str !== 'string'){
+        throw 'Invalid argument! it must be a string';
+    }
+    const allWords = str.split(' ')
+    const firstWord = allWords[0];
+    const lastWord = allWords[allWords.length-1];
```

Comment:
Did you consider when the given string has only one word? First and last words would be the same, I believe.

Reply:
Yes, it will break :(